pkg/actions/iam/get.go

```diff
@@ -0,0 +1,43 @@
+package iam
```

Comment:
shouldn't this be `actions/iamserviceaccounts` or is there overlap with other "resources"?

Reply:
yeah I did think about that, but the name felt too long. wdyt about `irsa`?